Assets/Scripts/DontDestroy.cs

```diff
@@ -5,13 +5,18 @@
 public class DontDestroy : MonoBehaviour
 {
     public GameManager manager;
+
     private int isFirstDay;
-    private void Awake() {
+    private void Awake()
+    {
         DontDestroyOnLoad(this.gameObject);
     }
-    private void Update() {
-        if (isFirstDay != 0) {
-            if (manager.sceneNum == 0) {
+    private void Update()
+    {
+        if (isFirstDay != 0)
+        {
+            if (manager.sceneNum == 0)
+            {
                 Destroy(this.gameObject);
             }
```

Comment:
If you're tracking number of days, you should name your variable `daysPlayed` or something, `isFirstDay` is more of a boolean type naming.